app/models/feedback.rb

```diff
@@ -10,9 +10,22 @@ class Feedback
     1 => 'Very dissatisfied'
   }
 
-  attr_accessor :email, :referrer, :user_agent, :comment, :rating
+  FEEDBACK_TYPES = {
+    feedback: [:rating, :comment, :email],
+    bug_report: [:event, :outcome, :email]
+  }
+
+  FEEDBACK_TYPES.keys.each do |type|
+    define_method "#{type}?" do
+      is?(type)
+    end
+  end
+
```

Comment:
wouldn't this be better as two separate hard coded methods rather than one dynamic one?

def feedback?
 FEEDBACK_TYPES[:feedback].include?(@type)
end

def bug_report?
 FEEDBACK-TYPES[:bug_report].include?(@type)
end